data/tr1/ship/cfg/TR1X_gameflow.json5

```diff
@@ -482,6 +485,7 @@
             "injections": [
                 "data/injections/backpack_cut.bin",
                 "data/injections/braid_cut2_cut4.bin",
+                "data/injections/font.bin",
```

Comment:
Do cutscenes not benefit from global injections?

Reply:
They have them disabled with `inherit_injections` set to false – most likely due to `O_LARA` not being present.